service/src/main/java/com/derpgroup/quip/bots/complibot/resource/CompliBotAlexaResource.java

```diff
@@ -101,22 +101,29 @@ public SpeechletResponseEnvelope doAlexaRequest(SpeechletRequestEnvelope request
     SpeechletResponseEnvelope responseEnvelope = new SpeechletResponseEnvelope();
     responseEnvelope.setSessionAttributes(sessionAttributesOutput);
 
+    SimpleCard card = new SimpleCard();
+    card.setContent(builder.getRawText());
+    card.setTitle("Alexa + Insultibot");
+    
+    SsmlOutputSpeech outputSpeech;
     // Create a VoiceOutput object with the SSML content generated by the manager
     if ((voiceInput.getMessageType() != MessageType.END_OF_CONVERSATION) && (voiceInput.getMessageType() != MessageType.STOP) && (voiceInput.getMessageType() != MessageType.CANCEL)) {
-      SimpleCard card = new SimpleCard();
-      card.setContent(builder.getRawText());
-      card.setTitle("Alexa + Complibot");
 
       @SuppressWarnings("unchecked")
       VoiceOutput<SsmlOutputSpeech> voiceOutput = (VoiceOutput<SsmlOutputSpeech>) VoiceMessageFactory.buildOutputMessage(builder.build(), InterfaceType.ALEXA);
-      SsmlOutputSpeech outputSpeech = voiceOutput.getImplInstance();
-      SpeechletResponse speechletResponse = new SpeechletResponse();
-      speechletResponse.setShouldEndSession(builder.isConversationEnd());
-      speechletResponse.setOutputSpeech(outputSpeech);
-      speechletResponse.setCard(card);
-
-      responseEnvelope.setResponse(speechletResponse);
+      outputSpeech = voiceOutput.getImplInstance();
+    }else{
+      outputSpeech = new SsmlOutputSpeech();
+      outputSpeech.setSsml("<speak></speak>");
+      card.setContent("Goodbye!");
```

Comment:
We need these to close the session. Right now all stop/cancel/aborts are holding the session open. The good news is that the ANOTHER intent works, and will repeat the non-closing CANCEL commands